packages/util/src/bytes.ts

```diff
@@ -73,6 +54,9 @@ export const bytesToBigInt = (bytes: Uint8Array, littleEndian = false): bigint =
   if (littleEndian) {
     bytes.reverse()
   }
+
+  // Ensure that bytes are actually bytes (and not just an array of numbers)
+  // bytes = Uint8Array.from(bytes)
```

Comment:
This is commented out?

Reply:
Meant to remove it entirely.  It breaks a bunch of VM tests and doesn't solve anything other than a couple of client tests that were already incorrect.